<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigGeneratorClass</class>
 <widget class="QMainWindow" name="ConfigGeneratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>100</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>388</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Parameters_Description_File"/>
    <addaction name="actionLoad_Parameters"/>
    <addaction name="actionSave_Parameters"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Parameters_Description_File">
   <property name="text">
    <string>Open Parameters Description File</string>
   </property>
  </action>
  <action name="actionLoad_Parameters">
   <property name="text">
    <string>Load Parameters</string>
   </property>
  </action>
  <action name="actionSave_Parameters">
   <property name="text">
    <string>Save Parameters</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>menubar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>ConfigGeneratorClass</receiver>
   <slot>menuBarAction(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>menuBarAction(QAction*)</slot>
 </slots>
</ui>
